NovelRT.Sdk/TemplateFiles/CMakeTemplate/src/PROJECT_NAME/CMakeLists.txt

```diff
@@ -1,32 +1,31 @@
-﻿include(CopyBuildProducts)
-include(CopyDirectoryRaw)
+﻿include(CopyDirectoryRaw)
 
 find_package(novelrt ${###PROJECT_NAME###_NOVELRT_VERSION} REQUIRED)
 
 set(SOURCES
     main.cpp
   )
 
-add_executable(###PROJECT_NAME### ${SOURCES})
+add_executable(###PROJECT_NAME###_Game ${SOURCES})

```

Comment:
Why have we appended `_Game` to the target like this?

Reply:
Targets must not be the same name as the project, therefore we're required to add _something_ to it if we're passing in the `###PROJECT_NAME###` variable here. 

We can change `_Game` to `_App`, etc. or we can add in a new variable that'll be replaced by C#.

Reply:
Ok, that's a problem hmmm. Maybe we just name the target `App` then and override the executable name in the cmake to be `###PROJECT_NAME###`?

Reply:
Fair enough (I think lol)

Reply:
We have a working example of this with the NovelRT DLL I think. :D